electron/main/llm/ipcHandlers.ts

```diff
@@ -38,4 +38,32 @@ export const registerLLMSessionHandlers = (store: Store<StoreSchema>) => {
     }
     await ollamaService.pullModel(modelName, handleProgress)
   })
+
+  ipcMain.handle('get-available-models', async () => {
+    try {
+      const models = await ollamaService.getAvailableModels()
+      return models
+    } catch (error) {
+      console.error('Error fetching available models:', error)
+      throw error
+    }
+  })
+
+  ipcMain.handle('delete-llm', async (event, modelName: string) => {
+    try {
+      const currentModels: LLMConfig[] = store.get(StoreKeys.LLMs) || []
```

Comment:
no need to complicate this ipc handler at all

it should literally only call the delete function from ollama and not check whether the model exists in the store or not

Reply:
Correct, resolved